dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -4273,6 +4273,23 @@ paths:
             uniqueItems: true
       security:
       - BEARER: []
+  /users/user/updateAcceptedDocuments:
```

Comment:
can probably just omit swagger for new endpoints. the openapi-only user response might be tricky though. if it's not tricky to only use openapi from the frontend, then remove this swagger